### DISCUSSION TITLE
It's just a dream language, right?

### DISCUSSION THREAD

**Discussion by victorting** [TodePond/GulfOfMexico#27] [Q&A] [answered]

**Comment by victorting** [comment]
So, where can i install it and use it. Haha

**Comment by TodePond** [comment, marked as answer]
you can install it with the installer, which you can install with the installer installer

or use the "Create DreamBerd App" app

**Reply by forgdeer** [reply]
but _where_ is it :(

**Reply by TodePond** [reply]
In the readme. If you're having trouble, take a look at the examples
